src/Controller/SidebarController.php

```diff
@@ -18,36 +18,55 @@ public function __construct(TranslatorInterface $translator)
 
     public function renderMenu(string $activeRoute): Response
     {
-        $menuItems = [
-            $this->generateMenuItem(
-                $this->translator->trans('Dashboard', [], 'sidebar'),
-                $this->translator->trans('Go to dashboard', [], 'sidebar'),
-                'default',
-                'tachometer-alt',
-                $activeRoute
-            ),
-            $this->generateMenuItem(
-                $this->translator->trans('Case list', [], 'sidebar'),
-                $this->translator->trans('Go to list of cases', [], 'sidebar'),
-                'case_index',
-                'list',
-                $activeRoute
-            ),
-            $this->generateMenuItem(
-                $this->translator->trans('Agenda list', [], 'sidebar'),
-                $this->translator->trans('Go to list of agendas', [], 'sidebar'),
-                'agenda_index',
-                'list-alt',
-                $activeRoute
-            ),
-            $this->generateMenuItem(
-                $this->translator->trans('Settings', [], 'sidebar'),
-                $this->translator->trans('Go to settings', [], 'sidebar'),
-                'admin',
-                'cog',
-                $activeRoute
-            ),
-        ];
+        if (!$this->isGranted('ROLE_BOARD_MEMBER')) {
```

Comment:
I think you should flip this if-else expression around to make it more clear that we're handling board member (and not non-board member) in a specific way, i.e.

```php
if ($this->isGranted('ROLE_BOARD_MEMBER')) {
  // Menu items for board member.
} else {
  // Menu items for non-board member.
}
```

Everywhere else in the code you use `$this->isGranted('ROLE_BOARD_MEMBER')` to do stuff for board members.

If we need to handle more roles in the future, rewriting to 

```php
$menuItems = [];
if ($this->isGranted('ROLE_CASEWORKER')) {
  $menuItems[] = // Dashboard
}
$menuItems[] = // Case list
…
if ($this->isGranted('ROLE_ADMINISTRATION')) {
  $menuItems[] = // Settings
}
```

may make it even more clear and remove some code duplication.

Reply:
Rewritten in 90e85a0251a5c6bd35b626e30960a25bf1e1e184 to the latter for ease of handling roles.